packages/alfa-rules/test/sia-r27/rule.spec.tsx

```diff
@@ -0,0 +1,134 @@
+import { h } from "@siteimprove/alfa-dom";
+import { None, Option } from "@siteimprove/alfa-option";
+import { test } from "@siteimprove/alfa-test";
+
+import R27, { Outcomes } from "../../src/sia-r27/rule";
+
+import { evaluate } from "../common/evaluate";
+import { oracle } from "../common/oracle";
+import { cantTell, failed, inapplicable, passed } from "../common/outcome";
+
+test(`evaluate() passes when some input rule passes`, async (t) => {
+  const target = <video src="foo.mp4" />;
+
+  const document = h.document([target]);
+
+  t.deepEqual(
+    await evaluate(
+      R27,
+      { document },
+      oracle({
+        "is-video-streaming": false,
+        "has-audio": true,
+        // R22
+        "has-captions": true,
+      })
+    ),
+    [passed(R27, target, { 1: Outcomes.HasTextAlternative })]
+  );
+});
+
+test(`evaluate() passes when some input rule passes`, async (t) => {
```

Comment:
Just spotted in this file as well:
```suggestion
test(`evaluate() passes when R31 passes`, async (t) => {
```